project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.2
```

Comment:
might have been possible to do the sbt PR separately, just to keep the size down?  But I realise some things tangle together a lot.